packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -170,7 +174,52 @@ export default class Editor {
      * @returns true if node is inserted. Otherwise false
      */
     public insertNode(node: Node, option?: InsertOption): boolean {
-        return node ? this.core.api.insertNode(this.core, node, option) : false;
+        let darkModeNormalize = null;
+        if (this.isDarkMode()) {
+            darkModeNormalize = this.convertContentToDarkMode(node);
+        }
+
+        const result = node ? this.core.api.insertNode(this.core, node, option) : false;
+
+        if (result && darkModeNormalize) {
+            darkModeNormalize();
+        }
+        return result;
+    }
+
+    private isElement(node: Node | HTMLElement): node is HTMLElement {
+        return (<HTMLElement>node).getElementsByTagName !== undefined && (<HTMLElement>node).style !== undefined;

```

Comment:
Does `instanceof HTMLElement`? not work?

[It appears to be a builtin class ](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement) so it should

Reply:
Yeah, you could use this, good catch.